feat(stable): added datamodelacl and dataModelInstancesacl to group type

EliasBjorne opened the pull request:
Added dataModelAcl and dataModelInstancesAcl to the Group type, as this was not available in the SDK

EliasBjorne committed:
Added datamodelAcl and dataModelInstancesAcl to Group type

EliasBjorne commented on packages/stable/src/types.ts:
```diff
@@ -83,6 +87,11 @@ export type AclAssets = Acl<AclActionAssets, AclScopeAssets>;
 
+export type AclDataModel = Acl<AclDataModel, AclScopeDataModel>;
+
+
```
```suggestion
```

EliasBjorne committed:
Update packages/stable/src/types.ts

EliasBjorne committed:
added missing WRITE_PROPERTIES

EliasBjorne committed:
ran regenerate

EliasBjorne committed:
linting

stefanmaretic commented on packages/stable/src/types.ts:
```diff
@@ -55,6 +55,10 @@ export type AclActionAssets = READ | WRITE;
 
 export type AclActionDataSets = READ | WRITE | OWNER;
 
+export type AclActionDataModelInstances = READ | WRITE;
```
Is this missing WRITE_PROPERTIES -- https://github.com/cognitedata/infrastructure/blob/master/lib/proto/api_acls/cogmono/api_acls/datamodelinstances_acl/datamodelinstances_acl.proto#L14?

stefanmaretic commented on packages/stable/src/types.ts:
```diff
@@ -55,6 +55,10 @@ export type AclActionAssets = READ | WRITE;
 
+export type AclActionDataModelInstances = READ | WRITE;
+
+export type AclActionDataModel = READ | WRITE | WRITE_PROPERTIES;
```
To me it looks like data models acl doesn't have `WRITE_PROPERTIES` -- https://github.com/cognitedata/infrastructure/blob/master/lib/proto/api_acls/cogmono/api_acls/datamodels_acl/datamodels_acl.proto#L12

EliasBjorne replied:
seems like I mixed those two up

EliasBjorne commented on packages/stable/src/types.ts:
```diff
@@ -55,6 +55,10 @@ export type AclActionAssets = READ | WRITE;
+export type AclActionDataModelInstances = READ | WRITE;
+
+export type AclActionDataModel = READ | WRITE | WRITE_PROPERTIES;
+
```
```suggestion
export type AclActionDataModelInstances = READ | WRITE | WRITE_PROPERTIES;

export type AclActionDataModel = READ | WRITE;

```

EliasBjorne committed:
Update packages/stable/src/types.ts

moved  | WRITE_PROPERTIES from AclActionDataModel to AclActionDataModelInstances

stefanmaretic reviewed (approved):
lgtm

BugGambit reviewed (approved):
LGTM, @EliasBjorne let's pay attention when merging to ensure it will bump the correct version (minor - `0.X.0`)
